knees4bees/dream-up-pup
# Feature/styling

### What does this PR do?
Adds styling and responsiveness.

### Are there any known bugs?
No.

### Summary of the changes
- Adds `fill` to the icon `.svg` files as needed so they'll show up on a dark background
- Puts custom dog-above-book icon on landing page
- Adds styling to all three pages
- Adds responsiveness to all three pages

### Notes for the reviewer
Keep an eye out for any lingering weirdness with the icons. If I forgot to change the color of any aspect, it might not show up (e.g. pawprint missing a toe, or something like that).

### How should this be tested?
Walk through the app and change the screen size on each view. Confirm that all icons are present and look okay, and that layout is acceptable at all screen sizes.
